io.github.fsobolev.TimeSwitch.json

```diff
@@ -0,0 +1,93 @@
+{
+    "app-id" : "io.github.fsobolev.TimeSwitch",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "timeswitch",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--talk-name=org.gnome.SessionManager",
+        "--system-talk-name=org.freedesktop.login1",
```

Comment:
you didn't explan why you need thes dbus permissions.

Reply:
The app allows to power off, reboot and suspend system. For that it uses org.gnome.SessionManager when running in GNOME, and org.freedesktop.login1 when running in another DE.